### DISCUSSION TITLE
Why was `main` renamed to `master`?

### DISCUSSION THREAD

**Discussion by astratagem** [nrdxp/nixdev#368] [Q&A] [answered]
I've seen a few references throughout the project to a `main` branch. However, no `main` branch exists. When I click a link referencing `main`, it redirects to `master` with a message indicating the branch had been renamed. I'm wondering why that is? Do you have plans to rename it back to `main`?

**Comment by blaggacao** [comment, marked as answer]
Afaik, it was renamed to `master` for tradition and as a vote against the fact that tech is politizicing around this.

I personally like that `main` is shorter and easier to type on my workman layout.

**Reply by nrdxp** [reply]
This was my fault. People can do whatever they want in their own projects, but I prefer to keep politics out of mine :slightly_smiling_face: 

I didn't realize it was ever changed to `main` in the first place otherwise I would have protested before the fact. I guess I was a bit upset that it was changed without my knowledge and acted hastily.

Also most official Nix projects still use master so it seems a bit more consistent with the community.

I should have probably updated the docs to reflect the change though :sweat_smile: 
I'll try to run through and update all the references soon unless you've got to them all already.

**Reply by astratagem** [reply]
Thanks for the clarification!
